TestingExamples/.gitignore

```diff
@@ -0,0 +1,363 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
I'm torn on how I feel about `.gitignore` files under source control. Sometimes I customize mine and don't want it marked as changed / needing to be updated upstream since you or others may want to customize your own.

Reply:
On the flip side, if you don't include the .gitignore in source control, you tend to end up with people accidently committing files they shouldn't be checking in because they don't have the same exclusions.

What customizations would you add though that you wouldn't want pushed to the repo? I would think that if you are wanting to exclude part of a project locally, those same files should be excluded for everyone.